waterdata/__init__.py

```diff
@@ -7,6 +7,9 @@
 
 # Loads configuration information from config.py and instance/config.py
 app.config.from_object('config')
-app.config.from_pyfile('config.py')
+try:
```

Comment:
Wondering if this is an appropriate thing to do... :thinking:

Reply:
I think it's fine. In the past, we always had tier specific settings, but perhaps we won't, particularly if we are using publically available endpoints

Reply:
I added a couple config settings for static assets - but you both will probably have feedback on how I set it up. I'm fine with this change though.

Reply:
@mbucknell -- Agreed. If we decide we need specific settings in the future, this doesn't preclude the use of a tier specific config.

Reply:
and there will be tier-specific config, even if it is just boring stuff like a dev-specific google analytics code